File: apps/dotnet-service/src/APIs/Payment/Dtos/PaymentCreateInput.cs
using DotnetService.Core.Enums;

namespace DotnetService.APIs.Dtos;

public class PaymentCreateInput
{
    public double? Amount { get; set; }

    public Booking? Booking { get; set; }

    public DateTime CreatedAt { get; set; }

    public string? Description { get; set; }

    public string? Id { get; set; }

    public string? Name { get; set; }

    public DateTime? PaymentDate { get; set; }

    public StatusEnum? Status { get; set; }

    public DateTime UpdatedAt { get; set; }
}

File: apps/hotel-booking-service-admin/src/api/payment/PaymentCreateInput.ts
import { BookingWhereUniqueInput } from "../booking/BookingWhereUniqueInput";

export type PaymentCreateInput = {
  amount?: number | null;
  booking?: BookingWhereUniqueInput | null;
  description?: string | null;
  name?: string | null;
  paymentDate?: Date | null;
  status?: "Option1" | null;
};
